components/tips/view.coffee

```diff
@@ -52,6 +55,9 @@ module.exports = class TipView extends Backbone.View
         @remove()
 
   remove: ->
-    Cookies.set @model.id, true
+    @dismisser.dismiss()
```

Comment:
So wait where is the code that prevents the tips from loading

Reply:
https://github.com/aredotna/ervell/blob/master/apps/user/tips.coffee#L3

Reply:
https://github.com/broskoski/ervell/blob/77cfbd74d8fa5b9523876942ef8f1a8ae20d3047/components/tips/helpers.coffee#L4

Reply:
Yeah, its a little weird. Tips get set on the route level (bc different tips for feed and for profile), so this adds all the tips that the route is trying to set, and rejecting the ones that already have cookies.

Reply:
It is odd. If it works it works. It'd probably be clearer to pass the dismissers into the templates or something, then use `if tip[id].dismisser.dismissed()`.

If this works for now that's fine as is.

Reply:
Yeah that would be clearer, personally I think we should replace these entirely with a different approach.

Reply:
Yeah we will. You can access local data like cookies through an Apollo datalink, setting up a schema for it and so forth. So we'll go that route when we switch these pages over.